rules/concepts.md

```diff
@@ -48,6 +48,154 @@ In some cases, it is useful to have a rule that is only manually triggered.
+Triggers define those events that, when they occur, causes the rule to run.
 
+There are the categories of rules that can be used to trigger a rule:
 
+| Event       | Description                                                                                                                  |
+|-------------|------------------------------------------------------------------------------------------------------------------------------|
+| **Items**   | Commands, updates, and changes on an individual Item.                                                                        |
+| **Groups**  | Groups are special Items that have other Items as members. Rules can be triggered on any Item event from any of its members. |
+| **Time**    | Rules can trigger based on specific times.                                                                                   |
+| **Channel** | Some Things have Channels that can trigger rules directly instead of being linked to Items.                                  |
+| **Thing**   | When Things change or update status they generate events (e.g. ONLINE, OFFLINE, etc.).                                       |
+| **System**  | Events that occur during important activities internal to openHAB itself, such as startup complete.                          |
+
+::: tip Triggers and Events - What's the difference?
+An event is something that happens and is detectable by openHAB.
+Some events for rules are time events, system events, Item state and command events, etc.
+
+A few examples of events that could be used to trigger a rule:
+- The time is midnight.
+- openHAB reached a runlevel 80 indicating Things are initialized.
+- The `Bedroom_Light` Item changed from OFF to ON.
+- A member of the `Lights` Group received a command.
+- Sunrise occurred based on an event Channel on an Astro Thing.
+- The Zwave controller Thing changed to OFFLINE.
+
+A Trigger is like a filter that, when it matches one or more events, causes the rule to run.
+Triggers can therefore be very specific to match very specific events (e.g. a change in a Switch Item from `ON` to `OFF`, a Dimmer Item receives `57` as a command, etc.)
+or relatively generic and match many events (e.g. every minute, a Color Item receives any type of command, a member of a Group Item changes in any way, etc.).
+:::
+  
+Here are the details for each trigger type:
+
+### **Items** Triggers
+
+You can listen to commands for a specific Item, on state updates or on state changes (an update might leave the state unchanged).
+You can decide whether you want to catch only a specific command/state or any.
+
+The `command` event means a `command`, which controls an Item (e.g. turns the light on) was sent.
+The `update` event means the Item's state got updated (e.g. the light turned to `ON`).
+
+**Item**-based triggers provide some information, e.g. the received command or the received state update, see [Available Values](#available-values).
```

Comment:
remove "-based". 

I assume the Available Values link is to be written?

Reply:
Yes, I am planning to write about the Available Values on this page (like you described in your plan in the issue) in the next days/weeks.